xbmc/video/VideoDatabase.cpp

```diff
@@ -584,22 +584,35 @@ void CVideoDatabase::CreateViews()
                  "    WHERE vv.idMedia = movie.idMovie "
                  "    AND   vv.media_type = '%s' "
                  "    AND   vv.itemType = %i "
-                 "  ) AS hasVideoExtras "
+                 "  ) AS hasVideoExtras, "
```

Comment:
hasVideoVersions and hasVideoExtras made 100% sense when the view was returning one row per movie.
Now that the view returns one row per movie and version combination, the requirement to have them is maybe not as strong.
Are they still needed in the C++ code? If yes the discussion ends there for v21. There will be time to revisit in v22 and to evaluate alternatives.

Reply:
hasVideoVersions is used at several paces. Let's keep it.
hasVideoExtras is currently not used and could be removed. Let's keep it, please. For consistency and in case we need it for Omega.